README.md

```diff
@@ -65,6 +65,35 @@ At the moment, no live-reload of certificates is supported, so to use new certif
 The TLS endpoint also supports TLS client authentication, which can be used to create sophisticated authentication and authorization structures easily.
 The intention is to have the TLS endpoint exposed to the inner application (your **backend**) and to have the WS endpoint exposed to your end users via a reverse proxy like nginx (which also serves the files for your **frontend**) which further separates TLS termination away from `autobahnkreuz` to your cloud provider/environment. The separation between TLS for internal usage and WS for external usage also yields a better privilege separation and therefore improves application security.
 
+## Logging Configuration
+Configuration of the `autobahnkreuz` Logger is supported by setting following environment variables:
+
+`SERVICE_LOGFORMAT`
+
+This allows setting the formatting of the Logger out of a set of predefined Formats:
+
+- `human`, or `debug`
+
+`%{color}[%{level:-8s}] %{time:15:04:05.000} %{longpkg}@%{shortfile}%{color:reset} -- %{message}`
+
+- `k8s`, `cluster`, or `machine`
+
+`[%{level:-8s}] %{time:2006-01-02T15:04:05.000} %{shortfunc} -- %{message}`
+
+Will default to `human` if none is set, and exit if the set value is not recognized.
+
+`SERVICE_LOGLEVEL`
+
+This allows setting the depth which the Logger will display at maximum. Values are defined as: 
+
+`CRITICAL`  
+`ERROR`  
+`WARN`  
+`INFO`  
+`DEBUG`  
+
```

Comment:
Maybe you want to introduce a list here.

Reply:
Included that in new version